articles/communication-services/quickstarts/email/includes/send-email-js.md

```diff
@@ -59,27 +59,30 @@ The following classes and interfaces handle some of the major features of the Az
-| EmailClientOptions  | This class can be added to the EmailClient instantiation to target a specific API version.                                                           |
-| EmailContent        | This class contains the subject and the body of the email message. The importance can also be set within the EmailContent class.                     |
-| EmailCustomHeader   | This class allows for the addition of a name and value pair for a custom header.                                                                     |
-| EmailMessage        | This class combines the sender, content, and recipients. Custom headers, attachments, and reply-to email addresses can optionally be added, as well. |
-| EmailRecipients     | This class holds lists of EmailAddress objects for recipients of the email message, including optional lists for CC & BCC recipients.                |
-| SendStatusResult | This class holds lists of status of the email message delivery.  
+| EmailClientOptions  | This interface can be added to the EmailClient instantiation to target a specific API version.                                                           |
+| EmailContent        | This interface contains the subject, plaintext, and html of the email message. |
+| EmailCustomHeader   | This interface allows for the addition of a name and value pair for a custom header.                                                                     |
+| EmailMessage        | This interface combines the sender, content, and recipients. Custom headers, importance, attachments, and reply-to email addresses can optionally be added, as well. |
+| EmailRecipients     | This interface holds lists of EmailAddress objects for recipients of the email message, including optional lists for CC & BCC recipients.                |
+| SendStatusResult | This interface holds the messageId and status of the email message delivery.
 
 ## Authenticate the client
 
- Import the **EmailClient** from the client library and instantiate it with your connection string. The code below retrieves the connection string for the resource from an environment variable named `COMMUNICATION_SERVICES_CONNECTION_STRING`. Learn how to [manage your resource's connection string](../../create-communication-resource.md#store-your-connection-string).
+Import the **EmailClient** from the client library and instantiate it with your connection string.
+
+The code below retrieves the connection string for the resource from an environment variable named `COMMUNICATION_SERVICES_CONNECTION_STRING` using the dotenv package. Use the `npm install` command to install the dotenv package. Learn how to [manage your resource's connection string](../../create-communication-resource.md#store-your-connection-string).
+
+```console
+npm install dotenv
+```
 
 Add the following code to **send-email.js**:
 
 ```javascript
 const { EmailRestApiClient } = require("@azure/communication-email");
```

Comment:
`EmailRestApiClient` throws an error. Use this instead: 

`const { EmailClient } = require("@azure/communication-email");`

Reply:
Good catch